DIMS-Core.Database/DIMS-Core.Database/Procedures/DeleteUser.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE PROCEDURE [dbo].[DeleteUser]
+	@UserId INT
+AS
+BEGIN
```

Comment:
Why have you added begin and end in the procedure?

Reply:
I have added BEGIN and END in the procedure to separate procedure's body from another code.
Like in tutorial:
[https://metanit.com/sql/sqlserver/11.1.php](url)